pa4/procedures.sql

```diff
@@ -0,0 +1,38 @@
+-- Procedure for retrieving employee details by employee ID
+DELIMITER $$
+CREATE PROCEDURE employee_details_by_id(IN emp_id INT)
```

Comment:
Please rename your procedures to <human readable name>_sproc (or simply _proc if you prefer this variant). So employee_details_by_id should be replaced with employee_details_by_id_sproc or employee_details_by_id_proc. This pattern should be applied to all you stored procedures.